integration/db/build.gradle

```diff
@@ -0,0 +1,87 @@
+import java.text.SimpleDateFormat
+
+configurations {
+    liquibase
+}
+
+dependencies {
+    api(group: "org.liquibase", name: "liquibase-core") {
+        version {
+            strictly liquibaseVersion
+        }
+    }
+    api(group: "org.liquibase.ext", name: "liquibase-hibernate6") {
+        version {
+            strictly liquibaseVersion
+        }
+    }
+
+    api 'org.springframework.boot:spring-boot-starter-data-jpa'
+    runtimeOnly 'com.microsoft.sqlserver:mssql-jdbc:11.2.3.jre17'
+    runtimeOnly 'com.h2database:h2'
+
+    liquibase 'com.microsoft.sqlserver:mssql-jdbc:11.2.3.jre17'
+}
+
+var driver = "com.microsoft.sqlserver.jdbc.SQLServerDriver"
+var username = "sa"
+var password = "P@ssw0rd000"
+var url = "jdbc:sqlserver://localhost:1433;trustServerCertificate=true;databaseName=F-PMS"
+
+tasks.register("setupDatabase") {
+    group = "hbc-database"
+
+    doFirst {
+        var masterFile = new File("${projectDir.path}/src/main/resources/master-changelog.yml")
+        if (!masterFile.exists()) {
+            masterFile.write(
+                "databaseChangeLog:\n" +
+                "  - includeAll:\n" +
+                "      path: changelogs/\n" +
+                "      relativeToChangelogFile: true"
+            )
+        }
+
+        var changeLogDir = new File("${projectDir.path}/src/main/resources/changelogs")
+        if (!changeLogDir.exists()) {
+            changeLogDir.mkdir()
+        }
+    }
+}
+
+tasks.register("createMigration", JavaExec) {
+    group = "hbc-database"
+    dependsOn([classes, setupDatabase])
+
+    classpath sourceSets.main.runtimeClasspath
+    mainClass = "liquibase.integration.commandline.Main"
+
+    args "--username=${username}"
+    args "--password=${password}"
+    args "--url=${url}"
+    args "--driver=${driver}"
+    args "--changeLogFile=src/main/resources/changelogs/changelog-${now()}.yaml"
+    args "--referenceUrl=hibernate:spring:${group}.integration.db" +
+            "?dialect=org.hibernate.dialect.SQLServerDialect"
```

Comment:
Apply physical_naming_strategy here, then we will no longer need to specify the column names.